src/FWlib/peripheral_sysctrl.h

```diff
@@ -129,6 +129,13 @@ typedef struct
     uint32_t source;     // bit combination of `SYSCTRL_WAKEUP_SOURCE_...`
 } SYSCTRL_WakeupSource_t;
 
+/**
+ * \brief Enable the PA, the Txen is set to GPIO2 and RXEN is set to GPIO6.
+ * This function should be called after the stack initialized. for example, at 
+ * the BTSTACK_EVENT_STATE of user_packet_handler.
```

Comment:
Add a note that this also needs to be called in `PLATFORM_CB_EVT_LLE_INIT` event.